src/autocomplete/databases/trino/grammar/TrinoParser.g4

```diff
@@ -852,6 +853,42 @@ identifier
     | DIGIT_IDENTIFIER_      # digitIdentifier
     ;
 
+catalogIdentifier
+    : identifier
+    ;
+
+schemaIdentifier
+    : catalogIdentifier DOT_ identifier
+    ;
+
+tableIdentifier
+    : schemaIdentifier DOT_ identifier
+    ;
+
+viewIdentifier
+    : tableIdentifier
+    ;
```

Comment:
Trino-client doesn't allow for `USE` statements, so we are forced to always give a fully qualified name like `catalog.schema.table`. This will greatly improve UX, because otherwise user will need to get an error from back end